Packs/Smokescreen_IllusionBLACK/Integrations/Smokescreen_IllusionBLACK/README.md

```diff
@@ -0,0 +1,541 @@
+Smokescreen IllusionBLACK is a deception based threat defence platform designed to accurately and efficiently detect targeted threats including reconnaissance, lateral movement, malware-less attacks, social engineering, Man-in-the-Middle attacks, and ransomware in real-time.
+This integration was integrated and tested with version xx of Smokescreen_IllusionBLACK
```

Comment:
please add the version of your API/Product here

Reply:
Version is added